tests/conftest.py

```diff
@@ -1,101 +1,47 @@
-import os
-
 import pytest
-import imp
 
 
-def pytest_addoption(parser):
-    default = 'scenario.py'
-    parser.addoption(
-        "--scenario",
-        action="store",
-        default=default,
-        help="YAML file defining scenarios to test. Currently defaults to: %s" % default
+@pytest.fixture()
+def CephNode(Ansible, Interface, Command, request):
+    vars = Ansible.get_variables()
+    node_type = vars["group_names"][0]
+    if not request.node.get_marker(node_type) and not request.node.get_marker('all'):
+        pytest.skip("Not a valid test for node type")
+
+    if request.node.get_marker("journal_collocation") and not vars.get("journal_collocation"):
+        pytest.skip("Skipping because scenario is not using journal collocation")
```

Comment:
Ok, I'll change the message.

Yeah, we probably will run into that situation but let's clean that up in a subsequent PR.